bin/get-amy.py

```diff
@@ -82,4 +89,17 @@ def sort_flags(data):
 
 
 if __name__ == '__main__':
-    main()
+    parser = argparse.ArgumentParser(description=sys.modules[__name__].__doc__)
```

Comment:
I would put all the argument handling code in a function that returns a triple (url, output_file, tags).